src/core/core.cpp

```diff
@@ -159,6 +160,14 @@ void wf::compositor_core_impl_t::init()
     });
     vkbd_created.connect(&protocols.vkbd_manager->events.new_virtual_keyboard);
 
+    protocols.vptr_manager = wlr_virtual_pointer_manager_v1_create(display);
+    vptr_created.set_callback([&] (void *data) {
+        auto event = (wlr_virtual_pointer_v1_new_pointer_event*) data;
+	auto ptr = event->new_pointer;
```

Comment:
indentation seems wrong here, maybe a mix of tabs and spaces? (should be 8 spaces here)